src/pomo/app.cljs

```diff
@@ -1,21 +1,43 @@
 (ns pomo.app
   (:require [re-frame.core :refer [dispatch-sync]]
+            [pomo.graphql]
+            [cljsjs.apollo-fetch]
+            [cljsjs.dataloader]
+            [cljsjs.graphql]
+            [district.ui.graphql]
             [reagent.core :as reagent]
             [reagent.dom :as rdom]
+            [mount.core :refer [defstate] :as mount]
             [pomo.events]  ;; These three are only
             [pomo.subs]    ;; required to make the compiler
             [pomo.views])) ;; load them
 
+(defonce ze-state (atom {}))
+
+(defstate app-core
+  :start (do
+           (println "Starting mount component in pomo.app")
+           (reset! ze-state {:state "Started"}))
+  :stop (do
+          (println "Stopping mount component in pomo.app")
+          (reset! ze-state {:state "Stop"})))
+
+(defn start-mount []
+  (-> (mount/with-args {:graphql {:schema pomo.graphql/schema :url "http://localhost:4567"}})
+      mount/start))
+
 (defn ^:dev/before-load stop []
-  (js/console.log "stop"))
+  (mount/stop)
+  (js/console.log "before-load stop"))
 
 (defn ^:dev/after-load start []
-  (js/console.log "start")
+  (js/console.log "after-load start")
+  (start-mount)
   (rdom/force-update-all))
 
 (defn ^:export init
   []
-  (dispatch-sync [:initialise-db])
-
   (println "Initializing pomo.app")
+  (dispatch-sync [:initialise-db])
+  (start-mount)
```

Comment:
Fails with
```
main.js:1673 Error: could not start [#'district.ui.graphql/district-ui-graphql] due to TypeError: Cannot set property 'parseValue' of undefined
    at eval (core.cljc:80)
    at mount$core$up (core.cljc:80)
    at Object.mount$core$bring [as bring] (core.cljc:247)
    at Function.eval [as cljs$core$IFn$_invoke$arity$variadic] (core.cljc:289)
    at Function.eval [as cljs$lang$applyTo] (core.cljc:281)
    at Function.eval [as cljs$core$IFn$_invoke$arity$2] (core.cljs:3947)
    at Function.eval [as cljs$core$IFn$_invoke$arity$variadic] (core.cljc:285)
    at Object.pomo$app$start_mount [as start_mount] (app.cljs:27)
    at Object.pomo$app$init [as init] (app.cljs:42)
    at eval (shadow.module.main.append.js:4)
```

![Screenshot from 2021-03-18 10-27-40](https://user-images.githubusercontent.com/674621/111603459-abcbaa80-87d4-11eb-8d54-05d91d5e7af6.png)

Reply:
After changing the dependency versions it gives
```
main.js:1673 Error: could not start [#'district.ui.graphql/district-ui-graphql] due to Error: Keyword must provide "serialize" function. If this custom Scalar is also used as an input type, ensure "parseValue" and "parseLiteral" functions are also provided.
    at eval (core.cljc:80)
    at mount$core$up (core.cljc:80)
    at Object.mount$core$bring [as bring] (core.cljc:247)
    at Function.eval [as cljs$core$IFn$_invoke$arity$variadic] (core.cljc:289)
    at Function.eval [as cljs$lang$applyTo] (core.cljc:281)
    at Function.eval [as cljs$core$IFn$_invoke$arity$2] (core.cljs:3947)
    at Function.eval [as cljs$core$IFn$_invoke$arity$variadic] (core.cljc:285)
    at Object.pomo$app$start_mount [as start_mount] (app.cljs:27)
    at Object.pomo$app$init [as init] (app.cljs:42)
    at eval (shadow.module.main.append.js:4)
```